numpy/lib/function_base.py

```diff
@@ -4154,13 +4154,34 @@ def _median(a, axis=None, out=None, overwrite_input=False):
         return mean(part[indexer], axis=axis, out=out)
 
 
-def percentile(a, q, axis=None, out=None,
-               overwrite_input=False, interpolation='linear', keepdims=False):
+def percentile(a, q, axis=None, out=None, overwrite_input=False,
+               interpolation='linear', keepdims=False):
     """
     Compute the qth percentile of the data along the specified axis.
 
     Returns the qth percentile(s) of the array elements.
 
+    Uses quantile() underneath, with
```

Comment:
This should be removed or modified. There does not need to be any implementation information in the docs. Instead, you could say something like "Calling `percentile` is identical to calling `quantile` with `weights` set to `None` and the percentiles divided by 100."